Sources/Sentry/SentryAsyncSafeLog.c

```diff
@@ -91,7 +91,6 @@ writeToLog(const char *const str)
             pos += bytesWritten;
         }
     }
-    write(STDOUT_FILENO, str, strlen(str));
```

Comment:
This was the line that did the unconditional log to console in file-only mode and double logged to console in debug mode.